.github/ISSUE_TEMPLATE/bug_report.md

```diff
@@ -1,11 +1,14 @@
 ---
 name: Bug Report
 about: Create a report to help us improve
+title: ''
+labels: ''
```

Comment:
What are these values for?

Reply:
the title of the issue and labels added. if we wanted any pre-added labels we could add them here.